src/actions/article.js

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+import dotenv from "dotenv";
+
+import {
+  CREATE_ARTICLE_SUCCESS,
+  CREATE_ARTICLE_FAILURE,
+  CREATE_ARTICLE_LOADING
+} from "../actionTypes/auth";
+
+dotenv.config();
+
+const setArticleSuccess = article => ({
```

Comment:
This should follow the naming convention `createArticle` `createArticleSuccess` `createArticleLoading` `createArticleError`.